golem-api-grpc/proto/golem/apidefinition/api_definition_registration_service.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package golem.apidefinition;
+
+import "golem/common/empty.proto";
+import "golem/template/template_id.proto";
+import "golem/apidefinition/api_definition_registration_error.proto";
+
+service ApiDefinitionRegistrationService {
```

Comment:
in relation to have shorter names 

do we need `Registration` in ApiDefinitionRegistrationService ?

it seems to me that it can be just `ApiDefinitionService`

similar for files, for example: api_definition_registration_error.proto -> api_definition_error.proto

Reply:
Yeah you have a point